Assets/Prefabs/Knitby.prefab

```diff
@@ -1,46 +1,46 @@
 %YAML 1.1
 %TAG !u! tag:unity3d.com,2011:
---- !u!1 &8090917524103177417
+--- !u!1 &657823012751151630
 GameObject:
   m_ObjectHideFlags: 0
   m_CorrespondingSourceObject: {fileID: 0}
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
   serializedVersion: 6
   m_Component:
-  - component: {fileID: 7720336601314418461}
-  - component: {fileID: 3351635507285597897}
-  - component: {fileID: 4476534854235773522}
-  - component: {fileID: 5072148313100846167}
-  m_Layer: 0
-  m_Name: Knitby
+  - component: {fileID: 4868919475267528974}
+  - component: {fileID: 5915865940897924245}
+  - component: {fileID: 2840611249138753973}
+  - component: {fileID: 4448341723275581311}
+  m_Layer: 7
+  m_Name: Visual
   m_TagString: Untagged
   m_Icon: {fileID: 0}
   m_NavMeshLayer: 0
   m_StaticEditorFlags: 0
   m_IsActive: 1
---- !u!4 &7720336601314418461
+--- !u!4 &4868919475267528974
 Transform:
   m_ObjectHideFlags: 0
   m_CorrespondingSourceObject: {fileID: 0}
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
-  m_GameObject: {fileID: 8090917524103177417}
+  m_GameObject: {fileID: 657823012751151630}
   serializedVersion: 2
-  m_LocalRotation: {x: 0, y: 0, z: 0, w: 1}
-  m_LocalPosition: {x: -2.41, y: 0.34, z: 0}
-  m_LocalScale: {x: 1, y: 1, z: 1}
```

Comment:
Knitby looks a little small to me- I tried increasing the scale to 1.2 and it looked more natural. If that also looks better to you we can change it